<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeBallParamsDlg</class>
 <widget class="QDialog" name="ChangeBallParamsDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_radio">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Radio</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_radio">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>44</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <number>10</number>
   </property>
   <property name="maximum">
    <number>200</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_radio_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Ball Color</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>red</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>blue</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>yellow</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>green</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>black</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>white</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_radio_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Speed x</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_radio_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Board Color</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>red</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>blue</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>yellow</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>green</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>black</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>white</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_radio_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Speed y</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_velo_x_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>65</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_velo_y_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>65</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ChangeBallParamsDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ChangeBallParamsDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
